public/locales/en/translation.json

```diff
@@ -177,6 +177,7 @@
             "title": "Override Cloud Sync Save folder"
         },
         "showfps": "Show FPS (DX9, 10 and 11)",
+        "start-in-tray": "Start in System Tray",
```

Comment:
I think Start Mimized is more commonly used on other apps so maybe its a better description. some don't even know what a tray is, believe me 😅

Reply:
Fixed!